### ISSUE TITLE
Error on reload

### ISSUE THREAD

**Issue by leon** [vercel/micro-dev#4]
I've installed the bare minimum micro and micro-dev

When I run micro-dev and change the code and save I get an error.

```
module.js:487
    throw err;
    ^

Error: Cannot find module '/Users/leon/Labs/micro-test/node_modules'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.resolve (internal/module.js:18:19)
    at Object.restartServer (/Users/leon/Labs/micro-test/node_modules/micro-dev/lib/listening.js:43:30)
    at Object.onceWrapper (events.js:314:30)
    at emitNone (events.js:105:13)
    at Server.emit (events.js:207:7)
    at emitCloseNT (net.js:1647:8)
    at _combinedTickCallback (internal/process/next_tick.js:135:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

It's when listening.js tries to remove the `require` cache that it fails.

```
// Remove file that changed from the `require` cache
  for (const item of toDelete) {
    const location = require.resolve(item)
    delete require.cache[location]
  }
```

The three items I'm getting:

```
item /Users/leon/Labs/micro-test
item /Users/leon/Labs/micro-test/index.js
item /Users/leon/Labs/micro-test/node_modules
```

What could be wrong?

**Comment by timneutkens** [comment]
This has been fixed in 1.0.1 please update 😄

**Comment by leon** [comment]
@timneutkens Found new problem.
I opened my project in intellij, which adds a `.idea` directory. Now it complains about that instead.

Should we ignore all `.` files and dirs?

**Comment by oskarrough** [comment]
Another observation: If I create any empty dir in the root, run `micro-dev`, restarting will crash as above. If I remove the empty folder again, it works. On 1.0.1.

**Comment by timneutkens** [comment]
Thanks for your feedback @leon @oskarrough ❤️ We'll address this very soon 👌

**Comment by danielyogel** [comment]
Error on reload: reproducible on 1.0.1

**Comment by leo** [comment]
Should be fixed in [1.0.2](https://github.com/zeit/micro-dev/releases/tag/1.0.2)!

**Comment by ansballard** [comment]
Confirmed my use case works, thanks!
